airbnb_project/airbnb_project/settings_test.py

```diff
@@ -0,0 +1,31 @@
+"""
+Django settings for airbnb_project project.
+
+Generated by 'django-admin startproject' using Django 5.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from .settings import *
+from dotenv import load_dotenv, find_dotenv
+import os
+
+load_dotenv(find_dotenv())
+
+# Override or add specific test settings here
+DATABASES['default']['NAME'] = 'airbnb_db'
```

Comment:
just curious. Are we lining this up with the database setup Tony is working on? I'm just concerned we'll end up with different instances of databases across the project.

Reply:
No, this is just a local db for testing purpose

Reply:
Unit test? If so, I think we could just mock data instead of having a whole database just for that, if that makes sense.

Reply:
> Unit test? If so, I think we could just mock data instead of having a whole database just for that, if that makes sense.

Thanks for your feedback, I looked into this and replaced creating test db with mocked models.